packages/components/NEW-COMPONENT-CHECKLIST.md

```diff
@@ -76,11 +76,9 @@ The engineering checklist has six parts: creating the feature branch, component
 ### Component Creation
 
 - [ ] create new branch from main for the component (`git checkout -b USER/COMPONENT-NAME`)
-- [ ] create new component
-  - `ember generate component hds/COMPONENT-NAME/index --gc` (the component won't need to be invoked with index, it's just to put all the files in the correct place)
-  - tip: if you forget to generate the backing class (via `--gc`), you can add it with `ember generate component-class hds/COMPONENT-NAME/index`.
-  - if it's a child component, then `hds/COMPONENT_NAME/CHILD_NAME` instead of `index`
-  - tip: if you need to destroy a component and start over again, `ember destroy component hds/COMPONENT-NAME/index` will remove the component and the related files as appropriate.
```

Comment:
question: does `destroy` also work with the blueprint generator? Have you tried it? I'd like to have the `destroy` tip still in there if possible.

Reply:
in which use case (for us, I mean) do you see the use of `destroy`? a developer can always reset a branch, undo the changes, revert a commit, etc.

Reply:
I use it all the time :)